### ISSUE TITLE
Mainline: GetItemButtonQuality is redefined in the deprecation fallbacks overwriting the usage in the ItemButton

### ISSUE THREAD

**Issue by plusmouse** [Stanzilla/WoWUIBugs#602]
The deprecation fallback CVar is enabled in EU, and it overwrites `GetItemQualityColor`, which is also defined in Interface/AddOns/Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua with a completely different return value, which then breaks the item button code when a conduit is passed in for display.

```lua
Message: .../Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua:294: attempt to index local 'color' (a number value)
Time: Wed Jul 24 00:11:16 2024
Count: 1
Stack: .../Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua:294: attempt to index local 'color' (a number value)
[string "=[C]"]: ?
[string "@Interface/AddOns/Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:294: in function `SetItemButtonOverlay'
[string "@Interface/AddOns/Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:249: in function <.../Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua:236>
[string "@Interface/AddOns/Blizzard_ItemButton/Mainline/ItemButtonTemplate.lua"]:672: in function `SetItemButtonQuality'
```

**Comment by Nevcairiel** [comment]
Don't worry, it'll fix itself with 11.0.2 as the deprecation layers have all been deleted.

**Comment by plusmouse** [comment]
Ok. Thanks :)

**Comment by Meorawr** [comment]
Fixed in 11.0.0.55824.
